splash/utils.py

```diff
@@ -84,19 +164,21 @@ def get_total_phymem():
     """ Return the total amount of physical memory available. """
     try:
         return psutil.virtual_memory().total
-    except AttributeError: # psutil < 2.0
+    except AttributeError:  # psutil < 2.0
         return psutil.phymem_usage().total
 
 
-def truncated(text, max_length=100, msg='...'):
+def truncated(text, max_length=100, msg=u'...'):
```

Comment:
I think it doesn't matter if we leave `msg='...'`, but it could make doctest work in 2.x and 3.x